DD2480-CI/src/main/java/mavenBuilder/MavenInvokerBuilder.java

```diff
@@ -3,14 +3,16 @@
 import java.io.File;
 import java.io.InputStream;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 
 import org.apache.maven.shared.invoker.*;
 
 public class MavenInvokerBuilder {
```

Comment:
You should probably add some documentation here just saying what the class does. Otherwise it looks good to me!